Modules/Filtering/FFT/itk-module.cmake

```diff
@@ -20,6 +20,7 @@ itk_module(ITKFFT
   TEST_DEPENDS
     ITKTestKernel
     ITKImageCompare
+    ITKImageIntensity
```

Comment:
Nitpick: this commit should have a `COMP: ` prefix, as it is a compile problem and not an actual code bug.

Reply:
Good catch, fixed.